src/main/java/com/faforever/client/game/KnownFeaturedMod.java

```diff
@@ -13,6 +13,7 @@
   FAF_DEVELOP("fafdevelop"),
   BALANCE_TESTING("balancetesting"),
   LADDER_1V1("ladder1v1"),
+  LADDER_2V2("ladder2v2"),
```

Comment:
Not a thing :). TMM games are played on the `faf` gamemode.

Reply:
If we create a QueueType object as I describe here https://github.com/FAForever/downlords-faf-client/pull/1786#discussion_r443147988
we will be able to remove all Ladder mentions in KnownFeaturedMod.